.github/workflows/trigger-caraml-doc-sync.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow triggers sync of docs to caraml-dev/docs
+
+name: Trigger CaraML Docs Sync
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  trigger-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Remote Doc Sync Workflow
+        uses: caraml-dev/docs/.github/actions/trigger-remote-docs-sync@main
+        with:
+          module: 'experiment'
+          git_https: 'https://github.com/caraml-dev/turing-experiments.git'
```

Comment:
Should be `https://github.com/caraml-dev/xp.git` now right?

Reply:
Good catch, let me fix that :)